AWSClientRuntime/Package.swift

```diff
@@ -24,16 +26,56 @@ let package = Package(
                 .product(name: "ClientRuntime", package: "ClientRuntime"),
                 .product(name: "AwsCommonRuntimeKit", package: "AwsCrt")
             ],
-            path: "./Sources"
+            path: "./AWSRuntime/Sources"
```

Comment:
1.  Seems like we should `AWSRuntime` should be changed to `AWSClientRuntime` ?
2. As a reminder to future you or future me, we will need to update `generatePackageSwift.swift` because these updates will break our release script.